### DISCUSSION TITLE
Dynamically reconfigure areas

### DISCUSSION THREAD

**Discussion by cerebrate** [jseidl/magic-areas#289] [Ideas / Feature Requests]
Specifically, it would be great to have a service available in Magic Areas that would cause it to discard its existing concept of which sensors make up an area and recompute them from the configuration, as it currently does during Home Assistant startup.

When adding sensors/reconfiguring/tweaking, this would make life a lot easier (and the process less disruptive) if you could check the effect of your changes on your areas without having to restart Home Assistant entire and disrupt all ongoing activities/reinitialize all other integrations.

**Comment by jseidl** [comment]
You can edit the area (without changing anything) all the way and it will re-load the area by the end of it. But indeed a service call to reload the area would be easier

**Comment by jseidl** [comment]
This can be done by manually reloading the config entry on the integrations page or by calling `homeassistant.reload_config_entry`. Closing!

**Reply by cerebrate** [reply]
Great stuff, thanks!
